client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx

```diff
@@ -89,19 +89,24 @@ const PremiumPlanDetails = ( { selectedSite, sitePlans, selectedFeature, purchas
 			/>
 
 			{ ! selectedFeature && (
-				<PurchaseDetail
-					icon={
-						<img alt={ translate( 'Customize Theme Illustration' ) } src={ customizeThemeImage } />
-					}
-					title={ translate( 'Customize your theme' ) }
-					description={ translate(
-						"You now have direct control over your site's fonts and colors in the customizer. " +
-							"Change your site's entire look in a few clicks."
-					) }
-					buttonText={ translate( 'Start customizing' ) }
-					href={ customizeLink }
-					target={ config.isEnabled( 'manage/customize' ) ? undefined : '_blank' }
-				/>
+				<>
+					<QueryBlogStickers blogId={ selectedSite.ID } />
```

Comment:
Does the parent component guarantee that `selectedSite` exists before rendering this component? Mainly that this value can be null in the global redux store on a clean browser cache (indexedDB/localStorage).